build.fsx

```diff
@@ -95,7 +99,14 @@ Target "Build" (fun _ ->
     |> Log "AppBuild-Output: "
 )
 
-// Build lots of packages for differet VM sizes
+let filesForZip =
```

Comment:
Format one-file-per-line so it's easier to see what's in the zip?

I suppose there's no other way to get this list of files....?